Portfolio setup

jlvFlores opened the pull request:
Probably spent way too long trying to understand how to use Flexbox. I used the second template from Figma and added an image imported from Figma to add to the headline section. Implemented working interactions to the only button shown, that being the one in the header. I used the default text and made sure to import the right fonts to make it look as much as I could make it identical to the original design. 

Please let me know if I missed anything required of this project.

jlvFlores committed:
Setup linters

jlvFlores committed:
Add header to HTML

jlvFlores committed:
Add header styles

jlvFlores committed:
Add headline section and image

jlvFlores committed:
Add styles for headline section

jlvFlores committed:
Update README with Microverse template

ABDULALI3468 commented on index.html:
```diff
@@ -0,0 +1,43 @@
+</head>
+<body>
+    <header>
+        <p>John Doe</p>
```
1- Please consider using this as a logo, not plain text. For that, you can export it from the respective [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27&t=dqLqteKgCvgwyC45-0)
2- Moreover, consider using a `a` tag with `href attribute` in it. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

ABDULALI3468 commented on index.html:
```diff
@@ -0,0 +1,43 @@
+<body>
+    <header>
+        <p>John Doe</p>
+        <button class="header-btn"><i class="fas fa-bars"></i></button>
```
1- Kindly use the `nav tag` instead of the `button tag` for this `hamburger icon`. The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices.

2- There exists a difference in the background and interactions of the `hamburger icon` I am attaching their respective images. Consider reviewing them.

![image](https://user-images.githubusercontent.com/94223367/212001160-87861064-405b-4d56-adf6-1f6922bf1973.png)
![image](https://user-images.githubusercontent.com/94223367/212001180-c08f204a-ecbc-4699-9334-97999d0e8b5e.png)

ABDULALI3468 commented on index.html:
```diff
@@ -0,0 +1,43 @@
+        <button class="header-btn"><i class="fas fa-bars"></i></button>
+    </header>
+
+    <div class="container">
```
Kindly consider using a `section tag` for the container instead of a `div tag`. It represents a generic standalone section of the document, so please use it. For more details and a deep dive into it consider going through the [following link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

ABDULALI3468 commented on index.html:
```diff
@@ -0,0 +1,43 @@
+        </ul>
+        <div class="export-image">
+            <img src="assets/images/illustration-header-mobile.png" alt="cool vectors">
+        </div>
```
Consider using a background image on the section tag(container) instead of `img tag`. (decorations are always background images, not `img` tags).

* If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

ABDULALI3468 commented on index.html:
```diff
@@ -0,0 +1,43 @@
+                <h1>Hey There. <br> I’m Alex</h1>
+            </li>
+            <li class="supporting-text">
+                <h2>I’am a Software Developer</h2>
```
The paragraph (e.g., "I'm a software ...") is a `p` tag. Consider replacing it with a `p tag`

* This way will be semantically correct and the screen readers and other technologies will thank you

huseyinbiyik replied:
[OPTIONAL] I think @ABDULALI3468 is confused with other templates. On this template, this area is the subtitle and you were right, it needs to be h2 element. 👍

ABDULALI3468 commented on index.html:
```diff
@@ -0,0 +1,43 @@
+                <h4>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</h4>
+            </li>
+            <ul class="social-media">
+                <li><i class="fa fa-brands fa-github"></i></li>
+                <li><i class="fa fa-brands fa-linkedin"></i></li>
+                <li><i class="fa fa-brands fa-twitter"></i></li>
+                <li><i class="fa fa-brands fa-twitter"></i></li>
+                <li><i class="fa fa-brands fa-medium"></i></li>
+            </ul>
+        </ul>
```
1- Consider adding a `a tag` in each of `li tags`. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href attribute`.

2- There exists a difference in the size of icons as compared to the respective [Figma template.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27&t=dqLqteKgCvgwyC45-0) Try making it closer to them.
![image](https://user-images.githubusercontent.com/94223367/211999962-e54afbf2-b27e-4bae-a7f3-74e4c0621389.png)
![image](https://user-images.githubusercontent.com/94223367/212000017-fbf701ef-bae9-4840-9e79-94d3b3f8e5e8.png)

ABDULALI3468 reviewed (changes requested):
# Changes Required 👍 

Hi @jlvFlores , 

<img src="https://media4.giphy.com/media/VlqzeaRnhCvPLfrSh6/giphy.gif"/>

You have done really great work on making the project according to the requirements 👏 and implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️
**_Check the comments below for the requested changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: ![GitHub followers](https://img.shields.io/github/followers/ABDULALI3468?label=ABDULALI&style=social)
> * This is My **Linkedin**: [![Linkedin Badge](https://img.shields.io/badge/-ABDUL%20ALI-blue?style=flat-square&logo=Linkedin&logoColor=white&link=https://www.linkedin.com/in/abdul-ali-5400bb216/)](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [![Twitter Badge](https://img.shields.io/badge/-@mrabdul_ali_-1ca0f1?style=flat-square&labelColor=1ca0f1&logo=twitter&logoColor=white&link=https://twitter.com/mrabdul_ali)](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jlvFlores committed:
Update and Fix header

jlvFlores committed:
Remove unused button style

jlvFlores committed:
Fix headline section content

jlvFlores commented:
I've followed the recommendations prompted to me by my previous CR to change the tags used for ones that are better suited, and removed the image that was in the HTML file and placed it as background using CSS.

huseyinbiyik commented on README.md:
```diff
@@ -1 +1,172 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```
[OPTIONAL] These placeholders are targeting you, not the visitors. So please replace them with your customized text. 👍

huseyinbiyik reviewed (approved):
Hi @jlvFlores,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Alt Text](https://media.giphy.com/media/wZjlCH43M3M0U/giphy.gif)



## Highlights
- Implementation of the design is brilliant 👨‍🎨
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝

## Optional suggestions
[OPTIONAL] I also recommend changing your Pull Descriptions to a list format like [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md), instead of a paragraph, so it will be more readable.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jlvFlores committed:
Fix tag under supporting text

jlvFlores committed:
Update README text

jlvFlores commented:
Improvements:
- Corrected the tag under supporting text
- Corrected the text of the placeholders in my README file
- Learned how to properly format pull request descriptions
